django_ipam/compat.py

```diff
@@ -10,12 +10,11 @@ def _is_subnet_of(a, b):
         try:
             # Always false if one is v4 and the other is v6
             if a._version != b._version:
-                raise TypeError("%s and %s are not of the same version".format((a, b)))
+                raise TypeError("%s and %s are not of the same version" % (a, b))

```

Comment:
doesn't this conflict with the other PR?

Reply:
Yes. It will, i'll close #68 for now and reopen when the work here is done. :smile:

Reply:
ok, one note for the future: you didn't have to close #68